protected/humhub/modules/file/migrations/m230618_135509_file_add_category_column.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * @link      https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license   https://www.humhub.com/licences
+ */
+
+use humhub\components\Migration;
+use humhub\modules\file\models\File;
+
+/**
+ * Add and film GUID column
+ */
+class m230618_135509_file_add_category_column extends Migration
+{
+    // protected properties
+    protected string $table;
+
+    public function __construct($config = [])
+    {
+        $this->table = File::tableName();
+        parent::__construct($config);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp(): void
+    {
+        $command = Yii::$app->getDb()
+                            ->createCommand()
+        ;
+
+        $schema = Yii::$app->getDb()->getTableSchema($this->table, true);
+        $after = $schema->getColumn('state') ? 'state' : 'guid';
+
+        $this->safeAddColumn(
+            $this->table,
+            'category',
+            $this->integer(11)
+                 ->unsigned()
+                 ->after($after)
+        );
+
+        $this->safeCreateIndex("ix-$this->table-category", $this->table, ['category', 'object_model', 'object_id']);
+
+        $command->update(
```

Comment:
I would leave this updates for now. Category should be `0` for the beginning.

Reply:
Since the default value shall be `0`, I've made the field non-nullable.